packages/mendel-core/tree-variation-walker.js

```diff
@@ -11,9 +11,9 @@ var MendelWalker = require('./tree-walker');
 util.inherits(MendelVariationWalker, MendelWalker);
 module.exports = MendelVariationWalker;
 
-function MendelVariationWalker(_lookupChains, _base) {
+function MendelVariationWalker(_lookupChains, _base, _hash) {
```

Comment:
it is used in super, this is pass through here, if we don't collect and someone calls without `new` it would break. Welcome to ES5 quirks, this is a detail of how to do what ES6 classes abstract for you.